stories/topics/con-azure-infrastructure-usage.adoc

```diff
@@ -20,7 +20,10 @@ Azure Database for PostgreSQL:: An Azure database service used for {PlatformName
 **Service Shape:** 1 TB
 Storage account:: The Azure service is used for file and block storage for local storage of projects, containers, etc.
```

Comment:
As per SG, I would avoid using "etc.", instead:

"The Azure service is used for file and block storage such as local storage of projects and containers."